README.md

```diff
@@ -139,5 +141,27 @@ var MailExampleApp = React.createClass({
 
 On Android, the `callback` will only be called if an `error` occurs. The `event` argument is unused!
 
+## Example 2
+Check if the user has an email account configured.
+
+```js
+var Mailer = require('NativeModules').RNMail;
+
+Mailer.canSend((error, canSend) => {
```

Comment:
You are right, but I wanted to stick to the error-first callback convention. I think this is more valuable than saving an argument.

What do you think?